example/image-fill-styles/index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+      <title>wavesurfer.js | Image Fill Styles Example</title>
+      <link href="data:image/gif;" rel="icon" type="image/x-icon" />
+      <!-- Bootstrap -->
+      <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" rel="stylesheet">
+      <link rel="stylesheet" href="../css/style.css" />
+      <link rel="stylesheet" href="../css/ribbon.css" />
+      <link rel="screenshot" itemprop="screenshot" href="https://katspaugh.github.io/wavesurfer.js/example/screenshot.png" />
+      <!-- wavesurfer.js -->
+      <script src="../../dist/wavesurfer.js"></script>
+      <!-- Demo -->
+      <script src="main.js"></script>
+      <!-- highlight.js for syntax highlighting in this example -->
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.2.0/styles/default.min.css">
+      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.2.0/highlight.min.js"></script>
+      <script>hljs.highlightAll();</script>
+      <style>#waveform {
+         z-index:-5;
+         margin-top:-150px;
+         }
+         .controls {
+         margin-top:-180px;
+         }
+      </style>
+   </head>
+   <body itemscope itemtype="http://schema.org/WebApplication">
+      <div class="container">
+         <div class="header">
+            <ul class="nav nav-pills pull-right">
+               <li><a href="/"><i class="glyphicon glyphicon-home"></i></a></li>
+            </ul>
+            <h1 itemprop="name">Image Fill Style</h1>
+         </div>
+         <p class="marketing">
+            Use any image element that has been loaded onto the page as the wave background.
+         </p>
+         <div id="demo">
+            <img width="100" src="water2.jpg" id="wavesurfer-water" style="display:none;" />
+            <img width="100" src="wood.jpg" id="wavesurfer-wood" style="display:none;" />
```

Comment:
where are these images from? they are not copyrighted are they?

Reply:
No, they are not copyrighted.   They are from Unsplash.  Attribution is not required, but appreciated.  I have an attribution list - I'll add it to the example page?

Robert Katzki
https://unsplash.com/photos/jbtfM0XBeRc

Julian Hochgesang
https://unsplash.com/photos/pVikeJwYOFA

Vino Li
https://unsplash.com/photos/osEBto6Mx8U

Jon Moore
https://unsplash.com/photos/5fIoyoKlz7A

Adam Azim
https://unsplash.com/photos/Ed2Fq0VXsCA

Hector Falcon
https://unsplash.com/photos/jXJ2_mRdlQY

Max Williams
https://unsplash.com/photos/_OoK2W7OPRM

Reply:
the audio files in the examples also have attribution at the bottom/footer of the example pages, i would suggest placing these for images there as well.

Reply:
done